metabase/metabase
# Make metabot setting do not depend on custom colors

Epic https://github.com/metabase/metabase/issues/22826

How to test:
- Run Metabase EE
- Go to Admin -> Settings -> Appearance
- Set a custom brand color and refresh the page
- Make sure that the `Metabot` setting is available and image is in the custom brand color
- Go to the home page and make sure that the `Metabot` logo is there and is in the custom brand color

Please note that we are going to modify the settings on the backend to return default values to match 0.43 behavior. 

<img width="557" alt="Screenshot 2022-07-12 at 20 40 07" src="https://user-images.githubusercontent.com/8542534/178557632-d18b4f42-1804-49cb-9293-4efddf275156.png">
<img width="935" alt="Screenshot 2022-07-12 at 20 40 18" src="https://user-images.githubusercontent.com/8542534/178557659-415397cb-8fb8-4a65-809f-d01e8747ef5a.png">
